### ISSUE TITLE
Single movie page: home btn hover effect

### ISSUE THREAD

**Issue by okalimbet** [Shawntru/Rotten-Tomatillos#6]
Users should be able to see a hover effect for a `Home` button

to let them know that their cursor is on a button they are about to click
